src/components/math-symbols/_math-symbol-icon-box.scss

```diff
@@ -0,0 +1,14 @@
+$subjectIconBoxBackground: $blueSecondary;
+$subjectIconBoxDarkerBackground: $bluePrimary;
+
+.sg-math-symbol-icon-box {
+  border-radius: 50%;
+  background-color: $subjectIconBoxBackground;
+  display: inline-block;
+  font-size: 0;
+  line-height: 0;
+
+  &--darker {
```

Comment:
`--dark` I don't think we us `darker` or `lighter` anywhere

Reply:
@aju we don't use it at all. I remove this file.